api/prontolista/test_instances/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+from django_extensions.db.models import TimeStampedModel
+
+from testcases.models import TestCase
+
+
+class TestInstance(TimeStampedModel):
+    testcase = models.ForeignKey(TestCase, on_delete=models.CASCADE)
+    assignee = models.CharField(null=True, blank=True, max_length=300)
+    statuses = (("passed", "Passed"), ("blocked", "Blocked"), ("failed", "Failed"))
```

Comment:
This should be upper case STATUSES

at first glance, I thought it was another field until I realize it's a list of choice

Reply:
Okie